### DISCUSSION TITLE
Anyone feels the new adapters converge slower and has lower sample per second compared to adapter-transformers?

### DISCUSSION THREAD

**Discussion by 14H034160212** [adapter-hub/adapters#728] [General]
Hi, 

Just a discussion that we performed some experiments by moving from adapter-transfomrers to adapters and we find out the training and inference speed is slower than the adapter-transformers. Does anyone have similar feeling? Does anyone have some suggestion? For example upgrading python version or something else.

**Comment by calpt** [comment]
Hey, thanks for bringing this up, we'll be investigating this! Do you have any experiments/ results you would be able to share?
